build-system/get-dep-graph.js

```diff
@@ -0,0 +1,280 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
```

Comment:
@cramforce since this is a verbatim copy paste, is this OK? or should I move this to third_party instead with your license and just modify (strip out anything not needed)?

Reply:
This is fine. I'm the sole author while I was a Google employee.